presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/file/FileHiveMetastoreConfig.java

```diff
@@ -32,9 +32,10 @@ public String getCatalogDirectory()
 
     @Config("hive.metastore.catalog.dir")
     @ConfigDescription("Hive file-based metastore catalog directory")
-    public void setCatalogDirectory(String catalogDirectory)
+    public FileHiveMetastoreConfig setCatalogDirectory(String catalogDirectory)
```

Comment:
as you are here. Should `catalogDirectory` be a `File`, and be annotated with`@FileExists`?

Reply:
Will extract separate PR for the cleanup (https://github.com/prestosql/presto/pull/5435) and fix it there